mcp-voice-agent/agent.py

```
@@ -0,0 +1,270 @@
+"""
+MCP voice agent that routes queries either to Firecrawl web search or to Supabase via MCP.
+"""
+
+from __future__ import annotations
+
+import asyncio
+import copy
+import json
+import logging
+import os
+from typing import Any, Callable, List, Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)